packages/ui-components/src/__tests__/StrategyPage.test.ts

```
@@ -95,50 +93,45 @@ describe('StrategySection', () => {
 
 		await waitFor(() => {
-			expect(screen.getByText('Error getting strategy details')).toBeInTheDocument();
-			expect(
-				screen.getByText("Cannot read properties of undefined (reading 'description')")
-			).toBeInTheDocument();
+			expect(screen.getByText('https://example.com/description.md')).toBeInTheDocument();
 		});
 	});
 
-	it('renders markdown if description is a markdown url', async () => {
+	it.only('renders markdown if description is a markdown url', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)